arboretum/common/kube_utils.py

```diff
@@ -0,0 +1,43 @@
+# -*- mode:python; coding:utf-8 -*-
+# Copyright (c) 2020 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Common Kubernetes functions."""
+
+from requests import HTTPError
+
+
+def get_cluster_resources(session, token, resource_types, verify=True):
```

Comment:
I'm curious why you chose to put this in a common util module.  Do you plan to reuse this code at some point?  Technically based on the [repo's guidelines](https://github.com/ComplianceAsCode/auditree-arboretum/blob/main/CONTRIBUTING.md#testing), code outside of a fetcher or a check requires unit tests to be written for it.  If you don't plan to reuse this code at any point then I suggest moving this function into the fetcher class as a method.  In addition to it being the right place for the logic, it will save you from having to write unit tests for it ;).

Reply:
I have a plan to reuse this function to implement IBM Cloud resource fetcher, and thus I'm now writing a unit-test code.